README.md

```diff
@@ -24,10 +24,11 @@ This service is written in [Scala](http://www.scala-lang.org/) and [Play](http:/
 
 ## Run the application
 
-To run the application execute: 
+To run the application execute 
+(values for routes,keys and feature switches can be found in the service manager config repository): 
 
 ```
-sbt 'run 9970' 
+sbt 'run 9970  <application_routes> <decryption_key> <api_key> <handoff_feature_switch>' 
```

Comment:
Could we get this a little closer to the actual command to run please?
e.g. run 9566 -Dapplication.router=testOnlyDoNotUseInAppConf.Routes (from VAT)

The only bit that can't be included is the encryption key itself, all of the other commands can.